### ISSUE TITLE
500 Error PRP-IP - PROD

### ISSUE THREAD

**Issue by mpawlowski85** [unicef/etools-partner-reporting-portal#1060]
Org: World Vision International
Workspace: Zimbabwe
1.	Go to PD: ZIM/PCA201829/PD201844
2.	Click on reports, then select QPR 1 or QPR2

Result: 500 Error. There is a toast message that says “An Error has occurred, please try again later. There is no info in the console. I only get this: 
![image](https://user-images.githubusercontent.com/26120371/49655221-15f72180-fa08-11e8-9e48-681801ae8852.png)

**Comment by sethgold8** [comment]
this is related to DB migration issue. Emailed Rob to see if we could reset the DB

**Comment by mpawlowski85** [comment]
It turns out it had nothing to do with the DB. 
The user has entered the Baseline as 7-30 (meaning 7h and 30 mins). As a result, PRP did not generate the reports. 
I have changed that in Django "reportables" to a dot "." in PRP and now the reports were generated. 

I have created a ticket for PMP to restrict the users from entering special characters in that field. 
https://app.clubhouse.io/unicefetools/story/9117/do-not-allow-special-characters-in-the-pd-indicator-baseline-target-fields
